### ISSUE TITLE
New Map: Overview becomes Map

### ISSUE THREAD

**Issue by jhudsonWA** [WSDOT/HATS#563]
- New map, try to match iPad view functionality. TOC, save visibility to user profile table. View only, no edits.

**Comment by ReedHunter-Wsdot** [comment]
Just to confirm, will this have all the Hats features?  Culverts, bridges, avalanche paths, geotechnical features, etc.?

**Comment by jhudsonWA** [comment]
Yep

Sent from my iPhone

On Nov 10, 2016, at 1:21 PM, Reed Hunter <notifications@github.com<mailto:notifications@github.com>> wrote:

Just to confirm, will this have all the Hats features? Culverts, bridges, avalanche paths, geotechnical features, etc.?

�

**Comment by ReedHunter-Wsdot** [comment]
Toc is now supporting visibility toggle for Hats layers.  AVL layer visibility toggle support will be next.

**Comment by joeschmit** [comment]
Is this checked in?

Sent from my iPhone

On Apr 19, 2017, at 1:48 PM, Reed Hunter <notifications@github.com<mailto:notifications@github.com>> wrote:


Toc is now supporting visibility toggle for Hats layers. AVL layer visibility toggle support will be next.

**Comment by jhudsonWA** [comment]
I’ll merge it in and push it to QA

On Apr 19, 2017, at 2:07 PM, Joe Schmit <notifications@github.com<mailto:notifications@github.com>> wrote:

Is this checked in?

Sent from my iPhone

On Apr 19, 2017, at 1:48 PM, Reed Hunter <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>> wrote:


Toc is now supporting visibility toggle for Hats layers. AVL layer visibility toggle support will be next.

**Comment by jhudsonWA** [comment]
It’s in QA now, may have to refresh cache if it’s not showing up so it pickers up the new client side code/images

On Apr 19, 2017, at 2:07 PM, Joe Schmit <notifications@github.com<mailto:notifications@github.com>> wrote:

Is this checked in?

Sent from my iPhone

On Apr 19, 2017, at 1:48 PM, Reed Hunter <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>> wrote:


Toc is now supporting visibility toggle for Hats layers. AVL layer visibility toggle support will be next.

**Comment by jhudsonWA** [comment]
Also layer choices won’t be saved yet between loads. Reed is waiting on me for some code.

On Apr 19, 2017, at 2:07 PM, Joe Schmit <notifications@github.com<mailto:notifications@github.com>> wrote:

Is this checked in?

Sent from my iPhone

On Apr 19, 2017, at 1:48 PM, Reed Hunter <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>> wrote:


Toc is now supporting visibility toggle for Hats layers. AVL layer visibility toggle support will be next.

**Comment by kechinn** [comment]
@ReedHunter-Wsdot  The Toc just supports visibility toggle for individual HATS features. Any way we can get the category level visibility toggle as well?

**Comment by ReedHunter-Wsdot** [comment]
That's part of what's left before I'd consider this task done:

- make visibility toggle buttons available for the AVL vehicles
- persist visibility choices between sessions
- visibility toggle buttons for the category panels
- Support for mouse click -> identify results for both Hats and Avl features

**Comment by ReedHunter-Wsdot** [comment]
Avl layer visibility can now be controlled from the Toc.  
Josh gave me a way to store between sessions which Hats layers you want, so I'm now working to plug that in.  I'll have a place to store Avl layer selections early next week, so those will be next.

**Comment by ReedHunter-Wsdot** [comment]
Visible layer choices are now being saved for Hats features.  What you turn on/off in the Toc is saved from one session to the next.  Once I have a place for storing Avl layer choices, I'll add support for saving those too.

**Comment by jhudsonWA** [comment]
@ReedHunter-Wsdot's last comment is now reflected in QA as well. The idea here is that any options selected to show, not show are persisted if the browser is closed and then brought back up.

**Comment by ReedHunter-Wsdot** [comment]
Saving of user selections is done now in the Toc.  
"filtering down by feature inventory results" is the only remaining feature for this task, but I'm not sure what that means.  @jhudsonWA has identified Avl map identify support as a higher priority task.

**Comment by kechinn** [comment]
@ReedHunter-Wsdot I'm assuming the filter down part refers to the ability to display on map what has been filtered by Feature Inventory, so display Feature Type by SRMP, Maintenance Area, Feature Number, Attributes, etc. Not as high of a priority as the identify support.

**Comment by jhudsonWA** [comment]
Yep. I'm going to remove that line from this task and we can add something new as a new task.

**Comment by jhudsonWA** [comment]
This went out with 5.4. Still needs feature identify support and upgrades. I'll open new issues/tasks for those and close this one as the main functionality is in place.
